src/bentoml/_internal/bento/build_config.py

```diff
@@ -511,6 +513,15 @@ def __attrs_post_init__(self):
     def is_empty(self) -> bool:
         return not self.requirements_txt and not self.packages
 
+    @functools.cached_property
+    def _jinja_environment(self) -> jinja2.Environment:
```

Comment:
can we centralized the template loading, So we don't have to initialize jinja env in multiple places

Reply:
and also the environment here is different from the other templates.

Maybe we can move to a centralized format in a different PR?